girder/wsgi.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2017 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import cherrypy
+from girder.utility import server
+
+cherrypy.config.update({'engine.autoreload.on': False,
+                        'environment': 'embedded'})
+cherrypy.config['server'].update({'cherrypy_server': False})
+
+application = server.setup()
+
+cherrypy.server.unsubscribe()
```

Comment:
It seems to be about determining who takes control of `SIGHUP` and `SIGTERM`. It also claims that it's dependent on which framework you're using. I'm opting to leave it out since their own uWSGI example (further down that page) does and I get a RuntimeWarning when adding it.